cli/src/main/kotlin/OrtServerMain.kt

```diff
@@ -47,6 +48,13 @@ class OrtServerMain : SuspendingNoOpCliktCommand(COMMAND_NAME) {
 
     init {
         subcommands(RunsCommand(ortServerConfig))
+
+        versionOption(
+            version = this::class.java.`package`.implementationVersion ?: "IDE-SNAPSHOT",
```

Comment:
Can the existing `ORT_SERVER_VERSION` constant be reused? If necessary, maybe my moving it to a more accessible place the CLI can depend on?